pkgs/by-name/op/openvpn3/patches/0001-customizable-asio-path.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/meson.build b/meson.build
+index 2bba337..092b4ce 100644
+--- a/meson.build
++++ b/meson.build
+@@ -68,7 +68,7 @@ endif
+ #
+ #  Setup additional include header dirs
+ #
+-asio_inc = get_option('asio_path') / 'asio' / 'include'
++asio_inc = get_option('asio_path') / 'include'
```

Comment:
Can you explain this part to me? I'm not really sure what this Meson syntax means, haha.

Reply:
`openvpn3-linux` allows overriding the directory from which `asio` dependency is taken (defaulting to submodule), but the unpatched path is too strict requiring `asio/` path for it (the syntax is just building a path). Thus I patch this part in order to allow using asio from nix-store.